containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
+    && apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+
+RUN mkdir -p ${APP_ROOT}
+COPY ./ ${APP_ROOT}/
+
+WORKDIR ${APP_ROOT}
+
+RUN gem install bundler
```

Comment:
Again, make the image more reproducible and ephemeral.
```suggestion
```

@pglombardo thoughts?

Reply:
I removed it. Necessary version of bundler will be installed within bundle install.
This is true as long as it is installed at all. And this should always be the case for ruby base images